src/main/java/com/google/devtools/build/lib/vfs/DigestHashFunction.java

```diff
@@ -20,21 +20,14 @@
 import com.google.common.collect.ImmutableSet;
 import com.google.common.hash.HashFunction;
 import com.google.common.hash.Hashing;
+import com.google.devtools.build.lib.hash.Blake3HashFunction;
 import com.google.devtools.build.lib.vfs.DigestHashFunction.DigestLength.DigestLengthImpl;
 import com.google.devtools.common.options.Converter;
 import com.google.devtools.common.options.OptionsParsingException;
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
 import java.util.HashMap;
 import java.util.Map.Entry;
 
-/**
- * Type of hash function to use for digesting files.
- *
- * <p>This tracks parallel {@link java.security.MessageDigest} and {@link HashFunction} interfaces
```

Comment:
i guess you'll be more likely to get questions about speed over here, but same thing--someone obviously wrote this code because they thought it mattered, is it easy to show that it doesn't?